with-react-query/search.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { useQuery } from "react-query"
+
+type User = {
+  name: string
+  followers: number
+}
+
+const Search = () => {
+  const [username, setUsername] = React.useState(undefined)
```

Comment:
Mainly to align with other example, we should import `useState` specifically:

```
  const [username, setUsername] = useState<string>()
```

Reply:
yes